Set output directories to be under the top level build directory
author: Nathan S. Watson-Haigh (University of Adelaide)
--- CMakeLists.txt	2017-07-07 09:45:07.233531339 +0930
+++ CMakeLists.txt	2017-07-07 11:30:12.274930847 +0930
@@ -10,10 +10,9 @@
 	set( SNIF_VERSION_BUILD 5 )
 ENDIF()
 
-
-set(EXECUTABLE_OUTPUT_PATH ${PROJECT_SOURCE_DIR}/bin/sniffles-core-${SNIF_VERSION_MAJOR}.${SNIF_VERSION_MINOR}.${SNIF_VERSION_BUILD}/)
-file(MAKE_DIRECTORY ${EXECUTABLE_OUTPUT_PATH})
-
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
 # Set a default build type for single-configuration
 # CMake generators if no build type is set.
